src/group/mls_group/create_commit.rs

```diff
@@ -221,10 +224,10 @@ impl PlaintextSecret {
         // Get a Vector containing the node indices of the direct path to the
         // root from our own leaf.
         let dirpath = treemath::leaf_direct_path(
-            provisional_tree.own_node_index().into(),
+            provisional_tree.own_node_index(),
             provisional_tree.leaf_count(),
         )
-        .expect("create_commit_internal: TreeMath error when computing direct path.");
+        .unwrap();
```

Comment:
This is still missing the comment on why the `unwrap()` is safe. I agree it's not part of the PR per-se, but let's finish this correctly.